packages/system/src/SVG.tsx

```diff
@@ -21,10 +23,8 @@ export const SVG: React.FC<SVGProps> = ({
   return jsx(
     'svg',
     {
-      width: size,
```

Comment:
@sebald this is the reason why the icons e.g. in checkbox have w and h 24. The default overrides.

Reply:
if width and height is outside the css prop it works

Reply:
https://github.com/marigold-ui/marigold/pull/1935